test/integration/hierarchy-change-auth.int.c

```diff
@@ -0,0 +1,231 @@
+#include "inttypes.h"
+#include "log.h"
+#include "sapi-util.h"
+#include "test.h"
+#include "sapi/tpm20.h"
+
+int test_0 (TSS2_SYS_CONTEXT *sapi_context);
+int test_1 (TSS2_SYS_CONTEXT *sapi_context);
+
+int tpm_hierarchies[] = {TPM_RH_OWNER, TPM_RH_PLATFORM};
+// TODO: TPM_RH_LOCKOUT, TPM_RH_ENDORSEMENT do not work in test_1
+
+int
+test_invoke (TSS2_SYS_CONTEXT *sapi_context)
```

Comment:
Move invoke test to the bottom and remove the forward declaration on test_0 and test_1